src/main/java/com/fasterxml/jackson/databind/cfg/BaseSettings.java

```diff
@@ -157,10 +162,10 @@ public final class BaseSettings
      */
 
     /**
-     * @since 2.16
+     * @since 2.19
      */
     public BaseSettings(ClassIntrospector ci, AnnotationIntrospector ai,
-            PropertyNamingStrategy pns, TypeFactory tf,
+            PropertyNamingStrategy pns, EnumNamingStrategy ens, TypeFactory tf,
```

Comment:
Due to backward-compatibility issue, Should write another constructor not modify the existing.
As you wrote comment below, write a new variant that takes {@link EnumNamingStrategy} instead.

```java
     * @deprecated Since 2.19, use variant that takes {@link EnumNamingStrategy} instead.
```

Reply:
The previous constructor was left unchanged, it's on [line 197](https://github.com/FasterXML/jackson-databind/pull/4723/files#diff-0e1933c5c67aaf7756c31c1a6faeb671597685b4444aa7e4cfa2b8f181359f7cR197).
This constructor is a new one with the additional parameter.

The change is just showed weirdly here

Reply:
Verified, true.